src/Pages/FindPeople.js

```diff
@@ -0,0 +1,90 @@
+import React, { useState, useEffect } from 'react';
```

Comment:
Have you tried this with any dummy data?  I see what you're doing here and I like the idea of returning the three most recently registered users if no match is found.  This is definitely something we could set up an api for.  I have a few questions though.

Reply:
I would do it on load not if there is no reasult. Does that make sense?